__tests__/cli.test.ts

```diff
@@ -0,0 +1,31 @@
+import { cli } from './utils'
+import { version } from '../package.json'
+
+test('prints version', async () => {
+  const output = await cli(['--version'])
+  expect(output).toContain(version)
+})
+
+test('prints help', async () => {
+  const output = await cli(['--help'])
+
+  expect(output).toContain(version)
+
+  for (const message of [
+    'Welcome to react-native-ci-cli',
+    'Quickly setup CI workflows for your React Native app',
+    '--help',
+    '--version',
+    '--skip-git-check',
+    '--preset',
+    'Use any combination of the following with --preset flag to specify your own set of workflows to generate',
+    '--lint',
+    '--jest',
+    '--ts',
+    '--prettier',
+    '--eas-update',
+    '--detox',
+  ]) {
+    expect(output).toContain(message)
```

Comment:
Isn't it something that should be done using snapshot tests?

Reply:
originally I wanted to test it with snapshots, but help message is colored and the way different environment print colored output is not consistent. For example, snapshot generated on my computer was not passing on github runner, for example here https://github.com/software-mansion-labs/react-native-ci-cli/actions/runs/10616060038/job/29425538720